Tests/Microsoft.AppCenter.Test.UWP/UWPPushTest.cs

```diff
@@ -153,5 +170,33 @@ public void ParseAppCenterPushMobileCenter()
             Assert.AreEqual(expectedResult.CustomData["key1"], actualResult.CustomData["key1"]);
             Assert.AreEqual(expectedResult.CustomData["key2"], actualResult.CustomData["key2"]);
         }
+
+        /// <summary>
+        /// Verify Push Service can be enabled or disabled correctly.  
+        /// </summary>
+        [TestMethod]
+        public void EnabledPushService()
```

Comment:
should we name it like VerifyPushServiceCanBeEnabledDisabledCorrectly. More over why this test is needed. You didn't added this functionality. Correct?

Reply:
I only added the EventHandler in `ApplyEnabledState` that will be triggered by `SetEnabledAsync` in this test. I found out that we don't have test that currently testing `ApplyEnabledState` so I just added this. @guperrot Should I remove this test?

Reply:
If we didn't have this setEnabled test before, then it's good that you added it. 

SetEnabledAsync is good enough as test name when it only tests the function of the same name. Now if you really want a longer name, go for it. But the current name does not fit well indeed.